crates/solidity/testing/snapshots/bindings_output/built_ins/address/generated/0.5.0-success.txt

```diff
@@ -65,3 +65,42 @@ References and definitions:
    │                                        │     
    │                                        ╰───── ref: built-in
 ───╯
+Definiens: 
+    ╭─[input.sol:1:1]
+    │
+  1 │ ╭─│ ▶ contract Test {
+  2 │ │ ╭─▶     function testAddress(address recipient) public {
+    │ │ │                            ────────┬────────  
+    │ │ │                                    ╰────────── definiens: 3
```

Comment:
WDYT of using `name: XXX` instead of `def: XXX`, to match the new API terminology? we would have `name`, `definiens`, and `ref`.

Reply:
Sounds good. I'll change it.